$OpenBSD: patch-configure,v 1.1 2009/01/06 10:16:52 ajacoutot Exp $

Do not use -fPIE, it gives empty unresolved symbols on powerpc.

--- configure.orig	Mon Jan  5 11:51:16 2009
+++ configure	Mon Jan  5 11:51:24 2009
@@ -20974,19 +20974,6 @@ if test "x$GCC" = "xyes"; then
      ;;
   esac
 
-  case " $CFLAGS " in
-  *[\ \	]-fPIE[\ \	]*) ;;
-  *) if cc_supports_flag -fPIE; then
-        PIE_CFLAGS="-fPIE"
-        if ld_supports_flag -z,relro; then
-           PIE_LDFLAGS="-pie -Wl,-z,relro"
-        else
-           PIE_LDFLAGS="-pie"
-        fi
-     fi
-     ;;
-  esac
-
   if test "x$enable_ansi" = "xyes"; then
     case " $CFLAGS " in
     *[\ \	]-ansi[\ \	]*) ;;
